pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -235,9 +236,15 @@ private static String getErrorMessage(HttpUriRequest request, CloseableHttpRespo
       controllerVersion = response.getFirstHeader(CommonConstants.Controller.VERSION_HTTP_HEADER).getValue();
     }
     StatusLine statusLine = response.getStatusLine();
+    String message = null;
+    try {
+      message = EntityUtils.toString(response.getEntity());
+    } catch (Exception e) {
+      // ignore
```

Comment:
Add an empty message or a generic message stating that unable to extract message.

Reply:
Fixed